admin-dev/themes/default/template/controllers/access/helpers/form/form.tpl

```diff
@@ -239,230 +239,285 @@
+                                                       data-parent="{$subsubChild.id_parent}" data-type="{$perm}"
+                                                       data-rel="{$access[$subsubChild.id_tab]['id_tab']}||{$profile.id_profile}||{$perm}||{$tabsize}||{count($tabs)}"
+                                                       class="ajaxPower {$perm} {$access[$subsubChild.id_tab]['id_tab']}"{if $access[$subsubChild.id_tab][$perm] == 1} checked="checked"{/if}/>
+                                              </td>
+                                            {else}
+                                              {assign var=has_all_rights value=false}
+                                              {assign var=enable_bulk_delete value=false}
+                                              <td>
+                                                <input type="checkbox"
+                                                       disabled="disabled"{if $access[$subsubChild.id_tab][$perm] == 1} checked="checked"{/if}/>
+                                              </td>
+                                            {/if}
+                                            {assign var=result_accesses value=$result_accesses + $access[$subsubChild.id_tab][$perm]}
+                                          {/foreach}
+                                          <td>
+                                            <input
+                                              type="checkbox"{if $access_edit == 1  && $has_all_rights} data-id="4" data-parent="{$subChild.id_parent}" data-type="all" data-rel="{$access[$subsubChild.id_tab]['id_tab']}||{$profile.id_profile}||all||{$tabsize}||{count($tabs)}" class="ajaxPower all {$access[$subsubChild.id_tab]['id_tab']}"{else} {assign var=enable_bulk_all value=false}class="all {$access[$subsubChild.id_tab]['id_tab']}" disabled="disabled"{/if}{if $result_accesses == 4} checked="checked"{/if}/>
+                                          </td>
+                                        </tr>
+                                      {/if}
+                                    {/foreach}
+                                  {/if}
+                                {/if}
                               {/foreach}
-                           {/if}
-                           </tbody>
-                        </table>
-                     </div>
-                  </div>
-               {else}
-                  <div class="col-lg-12">
-                     <div class="panel">
-                        {l s='Administrator permissions cannot be modified.' d='Admin.Advparameters.Notification'}
-                     </div>
-                  </div>
-               {/if}
+                            {/if}
+                          {/if}
+                        {/foreach}
+                      {/if}
+                    {/foreach}
+                  {/if}
+                  </tbody>
+                  <thead>
```

Comment:
I've moved the `<thead>` below the the `<tbody>` because, to know if we need to disable the bulk action, we need to browse all the lines otherwise it's not possible